spec/components/schemas/AccessPlan.yaml

```
@@ -51,22 +51,23 @@ allOf:
         type: boolean
         default: false
       sale_price:
-        description: Sale price.
+        description: Sale price.<br>
+          Only applies when `sale_enabled` is `true`.
         type: number
         format: float
         minimum: 0
         example: 1.00
```

The example object displays `1.00` as `1` which might be misleading at a glance.

```suggestion
        example: 99.97
```